.github/pull_request_template.md

```diff
@@ -1,30 +1,25 @@
-## Description & motivation
+## What are you changing in this pull request and why?
 <!---
-Describe your changes, and why you're making them. Is this linked to an open
-issue, a pull request on dbt core, etc?
+Describe your changes, and why you're making them. If linked to an open
+issue or a pull request on dbt core, then link to them here! 
 
 To learn more about the writing conventions used in the dbt Labs docs, see the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md).
 -->
 
-## To-do before merge
-<!---
-(Optional -- remove this section if not needed)
-Include any notes about things that need to happen before this PR is merged, e.g.:
-- [ ] Change the base branch
-- [ ] Ensure PR #56 is merged
+## Checklist
+<!--
+Uncomment if you're publishing docs for a prerelease version of dbt (delete if not applicable):
+- [ ] Add versioning components, as described in [Versioning Docs](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md)
+- [ ] Add a note to the prerelease version [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)
 -->
+- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [Versioning docs](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md) so my content adheres to these guidelines.
```

Comment:
We appear to have deleted or moved the versioningdocs.md document.

Reply:
D'oh! Good catch